Competition2017/src/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -47,10 +47,10 @@ public DriveSubsystem(WPIFactory factory, XPropertyManager propManager) {
         dVelProp = propManager.createPersistentProperty("Drive vel D", -100);
         fVelProp = propManager.createPersistentProperty("Drive vel F", 0);
         
-        leftDistanceProp = propManager.createEphemeralProperty("Left distance", 0);
-        rightDistanceProp = propManager.createEphemeralProperty("Right distance", 0);
+        leftEncoderTicksProp = propManager.createEphemeralProperty("Left encoder ticks", 0);
```

Comment:
Given that we'll also have left and right shooters, could we include the word "drive" somewhere in the property name?